tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -124,24 +130,39 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
 
         # Iterate through the files in a granule object
         for file in granule['files']:
-            source_name = file['name']
-            source_filepath = file['filepath']
-            if should_exclude_files_type(source_name, exclude_file_types):
+            file_name = file[FILE_FILENAME_KEY]
+            file_filepath = file[FILE_FILEPATH_KEY]
+            file_bucket = file[FILE_BUCKET_KEY]
+            file_source_uri = file[FILE_SOURCE_URI_KEY]
+
+            switch_file_to_output_format(file)
+
+            if should_exclude_files_type(file_name, exclude_file_types):
                 print(
-                    f"Excluding {source_name} from glacier backup "
+                    f"Excluding {file_name} from glacier backup "
                     f"because of collection configured {CONFIG_EXCLUDE_FILE_TYPES_KEY}.")
                 continue
-            copy_granule_between_buckets(source_bucket_name=file['bucket'],
-                                         source_key=source_filepath,
+            copy_granule_between_buckets(source_bucket_name=file_bucket,
+                                         source_key=file_filepath,
                                          destination_bucket=default_bucket,
-                                         destination_key=source_filepath,
+                                         destination_key=file_filepath,
                                          multipart_chunksize_mb=multipart_chunksize_mb)
-            copied_file_urls.append(file['filename'])
-            print(f"Copied {source_filepath} into glacier storage bucket {default_bucket}.")
+            copied_file_urls.append(file_source_uri)
+            print(f"Copied {file_filepath} into glacier storage bucket {default_bucket}.")
 
     return {'granules': granules_list, 'copied_to_glacier': copied_file_urls}
 
 
+def switch_file_to_output_format(file: Dict[str, Any]) -> None:
```

Comment:
Not sure the purpose of this. Wouldn't we follow the file schema Cumulus has put together?

Reply:
Task just specified input, so I thought I needed to preserve output. Fixed.